[2021-12-05 21:14:47,462] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2021-12-05 21:14:47,475] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2021-12-05 21:14:47,475] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-05 21:14:47,475] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-05 21:14:47,475] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-05 21:14:47,484] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): also_run_this> on 2021-12-05 00:00:00+00:00
[2021-12-05 21:14:47,487] {standard_task_runner.py:52} INFO - Started process 4777 to run task
[2021-12-05 21:14:47,491] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'scheduled__2021-12-05T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', '/Users/ballakeerthi/opt/anaconda3/envs/airflow/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/var/folders/b3/j_ybgjv53p33dy62wlktdr4m0000gp/T/tmpyhp_5o83', '--error-file', '/var/folders/b3/j_ybgjv53p33dy62wlktdr4m0000gp/T/tmp36xn6y0q']
[2021-12-05 21:14:47,493] {standard_task_runner.py:77} INFO - Job 6: Subtask also_run_this
[2021-12-05 21:14:52,520] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.also_run_this scheduled__2021-12-05T00:00:00+00:00 [running]> on host zipcodes-MacBook-Pro-3.local
[2021-12-05 21:14:57,551] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2021-12-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-05T00:00:00+00:00
[2021-12-05 21:14:57,551] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/b3/j_ybgjv53p33dy62wlktdr4m0000gp/T
[2021-12-05 21:14:57,552] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=scheduled__2021-12-05T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2021-12-05 00:00:00+00:00: scheduled__2021-12-05T00:00:00+00:00, externally triggered: False>"']
[2021-12-05 21:14:57,557] {subprocess.py:85} INFO - Output:
[2021-12-05 21:14:57,561] {subprocess.py:89} INFO - run_id=scheduled__2021-12-05T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2021-12-05 00:00:00+00:00: scheduled__2021-12-05T00:00:00+00:00, externally triggered: False>
[2021-12-05 21:14:57,561] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-05 21:14:57,576] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20211205T000000, start_date=20211206T021447, end_date=20211206T021457
[2021-12-05 21:14:57,623] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-05 21:14:57,642] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
